### ISSUE TITLE
When clicking on photos with the same name the first photo will pop up

### ISSUE THREAD

**Issue by wes3161** [nextcloud/photos#314]
**Describe the bug**
When clicking on photos with the same name the first photo will pop up instead of the one clicked

**To Reproduce**
1. Have multiple photos with the same name in different folders 
2. In the photos app click on any of the ones with the same name. No matter which one you click on the same photo will pop up

**Expected behavior**
The photo you click should pop up

**Comment by skjnldsv** [comment]
What do you mean, the same name?

**Comment by wes3161** [comment]
yeah, so if I have xyz.jpg in one folder and again xyz.jpg in another when I click on anyone one of those the same one will always pop up

Here's a gif showing it:
![photo](https://user-images.githubusercontent.com/22987969/80408879-e4e5ee00-8895-11ea-8ee0-f111cb94e293.gif)

**Comment by skjnldsv** [comment]
Thanks! :)
It make sense yes, moving to the viewer repository!
